commands/daemon.go

```diff
@@ -26,58 +25,59 @@ import (
 	"github.com/filecoin-project/sentinel-visor/schedule"
 )
 
-var log = logging.Logger("lily-cli")
-
 type daemonOpts struct {
 	api            string
 	repo           string
-	bootstrap      bool
+	bootstrap      bool // TODO: is this necessary - do we want to run visor in this mode?
```

Comment:
Running with bootstrap disabled will prevent Visor from connecting to peers (and therefore syncing the chain). It could prove useful in some future testing scenarios, but I'd be alright if we dropped this